src/index.js

```diff
@@ -9,10 +10,8 @@ const forge = require('node-forge')
 const protobuf = require('protocol-buffers')
 const path = require('path')
 
-const isNode = !global.window
-
 // protobuf read from file
-const messages = isNode ? protobuf(fs.readFileSync(path.resolve(__dirname, 'pb/crypto.proto'))) : protobuf(require('buffer!./pb/crypto.proto'))
+const messages = protobuf(fs.readFileSync(path.resolve(__dirname, '../protos/crypto.proto')))
```

Comment:
Because this way we can use `brfs` to always load the file. As soon as `js-ipfs` uses dignified.js it should work again, or you can manually add this config: https://github.com/dignifiedquire/dignified.js/blob/master/config/webpack.js#L89-L92